cpp/src/arrow/compute/kernels/vector_cumulative_ops_test.cc

```diff
@@ -37,19 +37,23 @@
 namespace arrow {
 namespace compute {
 
-constexpr static std::array<const char*, 6> kCumulativeFunctionNames{
+constexpr static std::array<const char*, 7> kCumulativeFunctionNames{
```

Comment:
Since we are changing this, can we simply make this a `std::vector<std::string>`? It will make further maintenance easier.

Reply:
Right, fixed. (Somehow I'm really obsessed with constexpr arrays......)